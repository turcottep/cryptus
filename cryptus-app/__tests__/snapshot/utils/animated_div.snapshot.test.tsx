import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

import AnimatedDiv from "../../../components/utils/animated_div";

describe("<AnimatedDiv />", () => {
  it("should display properly", async () => {
    const tree = renderer.create(<AnimatedDiv />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
