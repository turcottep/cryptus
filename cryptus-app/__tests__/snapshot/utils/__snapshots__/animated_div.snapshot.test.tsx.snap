// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AnimatedDiv /> should display properly 1`] = `
<div
  style={
    Object {
      "opacity": 0,
      "transform": "translateX(-100vw) translateZ(0)",
    }
  }
/>
`;
